briana_bennett/django/ninja_gold/ninjagold/views.py

```diff
@@ -0,0 +1,73 @@
+from django.shortcuts import render, redirect
+import random
+
+# Create your views here.
+def index(request):
+
+
+	return render(request, 'ninjagold/index.html')
+
+def process_money(request):
+
+	if request.method == 'POST':
+	
+		#add gold amount to gold in session
+		if 'gold' not in request.session:
+			request.session['gold'] = 0
+		if 'activities' not in request.session:
+			request.session['activities'] = []
+		
+		if request.POST['name'] == 'farm':
+			gold = random.randint(10,20)
+			request.session['gold'] += gold
+		
+		elif request.POST['name'] == 'cave':
+			gold = random.randint(5,10) 
+			request.session['gold'] += gold
+		
+		elif request.POST['name'] == 'house':
+			gold = random.randint(2,5)
+			request.session['gold'] += gold
+
+		
+		elif request.POST['name'] == 'casino':
+			gold = random.randint(-50,50)
+			request.session['gold'] += gold
+			
```

Comment:
if you had written ```request.session['gold'] += gold``` after long if/else chain then you would only have needed to write it once